### DISCUSSION TITLE
Any other way to visualize ??

### DISCUSSION THREAD

**Discussion by santoshpanda1995** [santoshpanda1995/Circular-Hough-Transform-using-MATLAB#6] [Q&A] [answered]
If anyone can suggest any other form of visualization , to circular though transform??

**Comment by shambhaveesrivastava12** [comment, marked as answer]
I am not very much into MATLAB, but according to me heatmaps and contour plots are very much useful for 2D visualizing as they allow to see how is data concentrated or dispersed across the 2D space, more specifically when the data is in a matrix format.

**Reply by santoshpanda1995** [reply]
Ok thanks will try...
